config/nginx/conf.d/ubersystem.conf

```diff
@@ -0,0 +1,31 @@
+# redirect any HTTP traffic to HTTPS
+server {
+    listen 80;
+    return 301 https://$host$request_uri;
+}
+
+# serve up SSL content, proxy to the ubersystem app
+server {
+    listen 443 ssl;
+
+    ssl on;
+
+    ssl_certificate             /etc/nginx/certs/uber.crt;
+    ssl_certificate_key         /etc/nginx/certs/uber.key;
+    ssl_session_cache           shared:SSL:10m;
+    ssl_session_timeout         5m;
+    ssl_protocols               SSLv3 TLSv1 TLSv1.1 TLSv1.2;
+    ssl_ciphers                 HIGH:!aNULL:!MD5;
+    ssl_prefer_server_ciphers   on;
+
+    index                       index.html index.htm index.php;
+
+    location / {
+        return 301 https://$host/uber;
+    }
+
+    location /uber/ {
+        proxy_pass          http://web:8282/uber/;
+        proxy_read_timeout  90;
+    }
+}
```

Comment:
ok cool then current version is solid.  and if we ever need to decide otherwise, it's easy to change.

and also, we could setup a different internal-only virtualhost that could still do SSL/client certs/whatever that isn't acceesible by the public